resources/generate_book.py

```diff
@@ -472,7 +472,7 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
             ),
         )),
         entry('pottery', 'Pottery', 'tfc:ceramic/vessel', pages=(
-            text('$(thing)Clay$() is an incredibly useful and balanced material, which can be used for pottery. However first, it needs to be located. Clay is usually hidden by grass, but it is found often in two locations. In areas with of at least 175mm $(l:the_world/climate#rainfall)Annual Rainfall$(), clay can be found in patches all over the place, however these patches are usually marked the by presence of certain $(thing)Plants$().'),
+            text('$(thing)Clay$() is an incredibly useful material which can be used for pottery. It can prove challenging to locate at first. Clay is usually hidden by grass, but it is often found in two locations. In areas with at least 175mm $(l:the_world/climate#rainfall)Annual Rainfall$(), clay can be found in patches all over the place, usually marked by the presence of certain $(thing)Plants$().'),
```

Comment:
Hey I wrote that joke and I like it. The other edits in this page are fine.

Reply:
I don't know which joke/edit you're referring to here, sorry!

Reply:
> Clay is an incredibly useful and balanced material,

Reply:
Oh, good, my first guess was correct!